README.md

```diff
@@ -25,6 +25,13 @@ Describe any dependencies that must be installed for this software to work.
 This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
 If specific versions of other software are required, or known not to work, call that out.
 
+- [NOAA-OWP/hydrofabric](https://github.com/NOAA-OWP/hydrofabric)
+  - Note that the arrow package needs `arrow::arrow_with_s3() == TRUE`. If `FALSE`, consider downloading arrow via [apache's r-universe](https://apache.r-universe.dev/arrow)
+  - Steps to install hydrofabric: Refer to wiki
+- [USGS nhdplusTools](https://github.com/doi-usgs/nhdplusTools/)
+- [pynhd](https://github.com/hyriver/pynhd)
+
+
 ## Installation - proc_fsds python package
 
 ### TLDR
```

Comment:
Note that this section has some inconsistencies, sometimes referring to the package as `proc_fsds`, but the package is actually called `fsds_proc`.

Reply:
@bolotinl - addressed in the latest commit.